pkg/motor/resource.go

```diff
@@ -142,6 +154,18 @@ func (r *motorResources) GetWhereIsByCreator(ctx context.Context, address string
 	for _, wi := range res.WhereIs {
 		r.whereIsStore[wi.Did] = &wi
 		ptrArr = append(ptrArr, &wi)
+
+		b := bucket.New(address, r.whereIsStore[wi.Did], r.shell, r.bucketQueryClient)
```

Comment:
Maybe we possibly try simplifying the creation of a bucket with just pkg/client?

Reply:
Thats probably a better idea, i can migrate some of this logic there with the merger of #528